service/history/historyEngine.go

```diff
@@ -81,41 +82,45 @@ import (
 const (
 	conditionalRetryCount                     = 5
 	activityCancellationMsgActivityNotStarted = "ACTIVITY_ID_NOT_STARTED"
+
+	historyServiceCallbackID = common.HistoryServiceName + "-%v"
```

Comment:
name is too generic. Maybe clusterInfoUpdateCallbackID

Reply:
updated